index.html

```diff
@@ -0,0 +1,121 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta name=”viewport” content=”width=device-width, initial-scale=1″ charset="UTF-8">
+    <link rel="stylesheet" href="css/style.css">
+    <link href='http://fonts.googleapis.com/css?family=Lato:300,400,500,600,700' rel='stylesheet'>
+    <link href='https://fonts.googleapis.com/css?family=Paytone One' rel='stylesheet'>
+    <title>Conference Page</title>
+  </head>
+  <body onload="initialExec();">
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.6.0/jquery.min.js"></script>
+    <script src="js/index.js"></script>
+    <div id="container">
+
+      <!-- header -->
+      <header id="header">
+        <div id="nav-banner">
+          <ul id="nav-banner-list">
+            <li id="nav-banner-list-item"><img id="icon-face" src="images/face.png"></li>
+            <li id="nav-banner-list-item"><img id="icon-twitter" src="images/twitter.svg"></li>
+            <li id="nav-banner-list-item">English</li>
+            <li id="nav-banner-list-item">My Page</li>
+            <li id="nav-banner-list-item">Logout</li>
+          </ul>
+        </div>
+        <div id="navigation">
+          <a id="navigation-logo" href="index.html"></a>
+          <ul id="navigation-list">
+            <li id="nav-navigation-list-item"><a href="index.html">Home</a></li>
+            <li id="nav-navigation-list-item"><a href="about.html">About</a></li>
+            <li id="nav-navigation-list-item"><a href="index.html">Program</a></li>
+            <li id="nav-navigation-list-item"><a href="index.html">Join</a></li>
+            <li id="nav-navigation-list-item"><a href="index.html">Sponsor</a></li>
+            <li id="nav-navigation-list-item"><a href="index.html">News</a></li>
+          </ul>
+          <div id="navigation-cccampaing">CC Campaign</div>
+        </div>
+        <nav id="hamburg-menu"><button id="hamburg-menu-button" type="button" onclick="hamburgClick()"></button></nav>
+      </header>
+
+      <!-- program -->
+      <section id="hero">
+        <div id="hero-superior">
+          <h2 id="hero-superior-title">"Hello! Sharing world"</h2>
+        </div>
+        <div id="hero-inferior">
+          <p id="hero-inferior-paraf">A joyful celebration believing in the value of openness and sharing, creating a positive change with people from over 80 countries is taking place in October, in Korea.</p>
+          <h4 id="hero-inferior-date">2015.10.15(THU)-16(FRI)</h4>
+          <h5 id="hero-inferior-mention">@ National Museum of Korea, Art Center Nabiand more</h5>
+        </div>
+      </section>
```

Comment:
- [ ] As mentioned by the previous reviewer to update your main hero text(`<h2 id="hero-inferior-title">CREATIVE COMMONS GLOBAL SUMMIT 2015</h2>`) to stick to the one of the Behance template, you should style it to look so and not take the screenshot of the template then use it as background image. Please, make sure you firstly write the hero text in your **HTML** file then style it correctly. 
To help you, just look for an image with those effects(e.g.: fire,...) then use that image as background of the hero text. You can refer this **[article](https://usefulangle.com/post/372/css-text-background-image)** to help you on how to do that.